ui-components/src/headers/SiteHeader.stories.tsx

```diff
@@ -1,171 +1,339 @@
 import * as React from "react"
 
-import { SiteHeader, NavbarDropdown } from "./SiteHeader"
-import { Button } from "../actions/Button"
-import { AppearanceSizeType } from "../global/AppearanceTypes"
+import { SiteHeader } from "./SiteHeader"
 
 export default {
   title: "Headers/Site Header",
 }
 
-export const standard = () => (
+export const basic = () => (
   <SiteHeader
+    homeURL={"/"}
+    // languages={[{ label: "English", onClick: () => console.log("Clicked English"), active: true }]}
     logoSrc="/images/logo_glyph.svg"
-    notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
+    // notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
     title="Alameda County Housing Portal"
-    skip="Skip to content"
-  >
-    <a href="#" className="navbar-item">
-      Hello World
-    </a>
-  </SiteHeader>
+    menuLinks={[]}
+  />
 )
 
-export const standardSlim = () => (
+export const withNotice = () => (
   <SiteHeader
+    homeURL={"/"}
     logoSrc="/images/logo_glyph.svg"
     notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
-    title="Housing Portal"
-    skip="Skip to content"
-    logoWidth={"slim"}
-  >
-    <a href="#" className="navbar-item">
-      Hello World
-    </a>
-  </SiteHeader>
+    title="Alameda County Housing Portal"
+    menuLinks={[]}
+  />
 )
 
-export const standardMedium = () => (
+export const withLanguage = () => (
   <SiteHeader
+    homeURL={"/"}
+    languages={[
+      { label: "English", onClick: () => console.log("Clicked English"), active: true },
+      { label: "Español", onClick: () => console.log("Clicked Español"), active: false },
+      { label: "中文", onClick: () => console.log("Clicked 中文"), active: false },
+    ]}
     logoSrc="/images/logo_glyph.svg"
-    notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
     title="Alameda County Housing Portal"
-    skip="Skip to content"
-    logoWidth={"medium"}
-  >
-    <a href="#" className="navbar-item">
-      Hello World
-    </a>
-  </SiteHeader>
+    menuLinks={[]}
+  />
 )
 
-export const standardWide = () => (
+export const withLanguageAndNotice = () => (
```

Comment:
would it be possible to test withLanguageAndNotice with the notice on mobile view too? I realize that makes a lot of variations though. Would it make sense to set up knobs for this component because there are so many props?